source/install.txt

```diff
@@ -25,6 +24,21 @@ Select your operating system:
          - 64-bit version of macOS 10.12 or later.
          - MongoDB 3.6 or later.
 
+         |compass-short| supports x64 and ARM64 architectures.
+         
+         M1 Support
+         ~~~~~~~~~~
+
+         You can run |compass| on M1 platforms.
+         
+         - If you run an operating system prior to macOS Big Sur, to run
```

Comment:
Consider putting the newer operating system first, overtime fewer users will need to read both blurbs.

Reply:
Updated.